Automattic/wp-calypso
# Annotate imports as "External dependencies" or "Internal dependencies"

#### Changes proposed in this Pull Request

Fixes the rule `wpcalypso/import-docblock` across the whole repo

#### Testing instructions

* Run `./node_modules/.bin/eslint-nibble --ext .js,.jsx,.ts,.tsx,.md.js,.md.javascript,.md.jsx .` (it takes a few minutes) and verify there are no more errors for `wpcalypso/import-docblock`
* Ensure tests pass, ignore lint errors.
